hri_safety_sense/include/VscProcess.h

```diff
@@ -71,6 +80,9 @@ namespace hri_safety_sense {
 		  ros::Timer 	  		mainLoopTimer;
 		  ros::ServiceServer    estopServ, keyValueServ, keyStringServ;
 		  ros::Publisher		estopPub;
+		  ros::Subscriber 		vibrate_src_Sub;
+		  ros::Subscriber 		display_src_on_Sub;
+		  ros::Subscriber 		display_src_off_Sub;
```

Comment:
```suggestion
		  ros::Subscriber 		vibrateSrcSub;
		  ros::Subscriber 		displaySrcOnSub;
		  ros::Subscriber 		displaySrcOffSub;
```
I don't like it, but we should try to match the style used in this code. Consistency is a huge part of maintiability and readability.